test/sample-test/run_test.sh

```diff
@@ -152,6 +152,7 @@ if [ "$TEST_NAME" == 'tf-training' ]; then
     sed -i -e "s|gcr.io/ml-pipeline/ml-pipeline-dataflow-tf-predict:\([a-zA-Z0-9_.-]\)\+|${DATAFLOW_PREDICT_IMAGE}|g" kubeflow-training-classification.py
     sed -i -e "s|gcr.io/ml-pipeline/ml-pipeline-local-confusion-matrix:\([a-zA-Z0-9_.-]\)\+|${LOCAL_CONFUSIONMATRIX_IMAGE}|g" kubeflow-training-classification.py
   fi
+  sed -i -e "s|use_gpu\s*=\s*False|use_gpu = True|g" kubeflow-training-classification.py
```

Comment:
That seems a bit hacky. Could we use proper parameter passing instead of using sed on a Python file?

Reply:
We did sed replace all over the places in the test script. There is currently a limitation that user cannot pass gpu setting as pipeline parameter as gpu must be set before the dsl compile time. That's why we cannot use parameter to control this test.

Reply:
OK, thanks.